Formula/grakn.rb

```diff
@@ -0,0 +1,40 @@
+class Grakn < Formula
+  desc "The Database for AI"
+  homepage "https://grakn.ai"
+  url "https://github.com/graknlabs/grakn/releases/download/v0.12.1/grakn-dist-0.12.1.tar.gz"
+  sha256 "3658138ccd74e51a0072a42f5e4fff43763506b02e581b572b05f50b1c63eb72"
+
+  depends_on :java => "1.8+"
+
+  def install
+    # Install jars in libexec to avoid conflicts
+    libexec.install Dir["*"]
+
+    # Symlink binaries
+    bin.install Dir["#{libexec}/bin/*"]
+
+    # Set where the executable is
+    bin.env_script_all_files(libexec/"bin", :CASSANDRA_HOME => ENV["CASSANDRA_HOME"])
+  end
+
+  def caveats; <<-EOS.undent
+    Database data is at: #{var}/grakn/db/
+    Configuration files are at: #{etc}/grakn/config/
+
+    To start Grakn:
+    $ grakn.sh start
+
+    To open the REPL:
+    $ graql.sh
+
+    Web UI:
+    http://localhost:4567
+
+    Data should not be affected on upgrade.
+    EOS
+  end
```

Comment:
You can remove all these caveats as they are mostly not Homebrew specific, thanks!